modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/servlet/CacheControlFileEntryHttpHeaderCustomizer.java

```diff
@@ -51,8 +51,8 @@ private String _getHttpHeaderValue(FileEntry fileEntry, String currentValue)
 		throws PortalException {
 
```

Comment:
Hey @RenanUmV, just remove the `configurationPid` from this class. It is not needed. Thx